report.c

```diff
@@ -406,14 +406,14 @@ void calculate_motor_voltage(){
 
   for (i=0;i<(VOLTAGE_SENSOR_COUNT-1); i++){
     ADCSRB &= ~(1<<MUX5_BIT); // Clear MUX5_BIT which is set when force sensor is target
-    ADMUX = (1<<REFS0) + voltage_result_index; // set next motor target for ADC
```

Comment:
`voltage_result_index` was used when I had the ADC on an ISR. But in this situation `voltage_result_index` doesn't get incremented while `i` does. I should be using `i` in this situation otherwise I will be printing the voltages of X,Y,Y,Y,F instead of X,Y,Z,C,F.